src/js/browser-plugin.js

```diff
@@ -1217,20 +1224,47 @@ const createLabelModulesExplainabilityContent = (label, modules) => {
   if (!auxLabel) auxLabel = label;
   let textHtml = browserAPI.i18n.getMessage('content_tagged_by__html', [auxLabel, Object.keys(modules).length]);
   explainInfoText.innerHTML = textHtml;
-  let spaceTxt = document.createTextNode(" ");
-  explainInfoText.append(spaceTxt);
-  let specificallyTxt = document.createTextNode(browserAPI.i18n.getMessage('specifically'));
-  explainInfoText.append(specificallyTxt);
   explainInfoContent.append(explainInfoText);
-  let explainInfoList = document.createElement("UL");
-  if (modules) {
+
+  let explainInfoPart2 = document.createElement("SPAN");
+  let explainInfoPart2Txt1 = document.createTextNode(browserAPI.i18n.getMessage('modules_check_description'));
+  explainInfoPart2.append(explainInfoPart2Txt1);
+  let notVerifiableTxt = browserAPI.i18n.getMessage('not_verifiable');
+  let explainInfoPart2Txt2 = document.createTextNode(browserAPI.i18n.getMessage('not_verifiable_case_description', notVerifiableTxt));
+  explainInfoPart2.append(document.createTextNode(" "));
+  explainInfoPart2.append(explainInfoPart2Txt2);
+  explainInfoContent.append(document.createElement("BR"));
+  explainInfoContent.append(explainInfoPart2);
+
+  let explainInfoPart3 = document.createElement("SPAN");
+  let explainInfoPart3Txt = browserAPI.i18n.getMessage('more_generic_analysis_info__html', CoinformConstants.ANALYSIS_INFO_URL);
```

Comment:
Two '__' between 'info' and 'html'. Intended or misspelling?

Reply:
Yep, is intended. My way to separate plain text messages than the ones that allows html tags. As we also use underscores in the messages keys, I decided to separate with the "html" notation using 2*underscores.